wsdl/ver10/device/wsdl/devicemgmt.wsdl

```diff
@@ -2159,6 +2164,32 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 							<xs:documentation>User credential for the storage server</xs:documentation>
 						</xs:annotation>
 					</xs:element>
+					<xs:element name="ConfigurationRenewal" minOccurs="0">
+						<xs:complexType>
+							<xs:sequence>
+								<xs:element name="RenewalEndpoint" type="xs:anyURI">
+									<xs:annotation>
+										<xs:documentation>Remote URL to be queried by the device to renew the storage configuration.</xs:documentation>
+									</xs:annotation>
+								</xs:element>
+								<xs:element name="AuthorizationServer" type="tt:ReferenceToken">
+									<xs:annotation>
+										<xs:documentation>JWTConfiguration token referring to an Authorization server that provides JWT token to authorize with the the renewal endpoint.</xs:documentation>
```

Comment:
Its ambiguous if you are referring to token here as **AuthorizationServerConfiguration** Token or **JWTConfiguration** as element name and documentation seems a bit out of sync?

Reply:
Good point, it should have been AuthorizationConfiguration token. Fixed!